packages/paste-core/components/avatar/src/index.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from 'react';
+import {Text} from '@twilio-paste/text';
+import {Box, safelySpreadBoxProps} from '@twilio-paste/box';
+import {IconSize} from '@twilio-paste/style-props';
+import {getComputedTokenNames} from './utils';
+
+export interface AvatarProps {
+  size: IconSize;
+}
+export interface AvatarImageProps {
+  src: string;
+  alt: string;
+}
+
+const AvatarContext = React.createContext<AvatarProps>({size: 'sizeIcon10'});
+
+export const AvatarImg: React.FC<AvatarImageProps> = ({src, alt, ...props}) => {
+  const {size} = React.useContext(AvatarContext);
+  return <Box {...safelySpreadBoxProps(props)} as="img" alt={alt} maxWidth="100%" src={src} size={size} />;
+};
+
+const Avatar: React.FC<AvatarProps> = ({children, size = 'sizeIcon70'}) => {
+  const computedTokenNames = getComputedTokenNames(size);
+  return (
+    <AvatarContext.Provider value={{size}}>
+      <Box
+        as="div"
+        backgroundColor="colorBackgroundPrimaryLight"
+        borderRadius="borderRadiusCircle"
+        overflow="hidden"
+        size={size}
+      >
+        <Text
```

Comment:
Currently this wraps the image component if it's passed as a child.  What if we check if `typeof children === 'string'` and wrap it in a Text then?

Reply:
Yeah, we could do that. Do you think it's worth the conditional check?

Reply:
Now that we don't have a child image component, it makes a bunch of sense to conditionally render the text